### ISSUE TITLE
ParseInstallation not being saved in Keychain

### ISSUE THREAD

**Issue by lsmilek1** [parse-community/Parse-Swift#235]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/Parse-Swift/security/policy).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/main/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/Parse-Swift/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the latest versions of [Parse Server](https://github.com/parse-community/parse-server/releases) and the [Parse Swift SDK](https://github.com/parse-community/Parse-Swift/releases). 

### Issue Description

ParseInstallation.current found not nil only after a fresh install (cleared keychain) and on further app runs found always nil.

### Steps to reproduce

0) clear simulator or device keychain
1) run ParseSwift initialisation in `didFinishLaunchingWithOptions` and print `ParseInstallation.current`
```swift
   func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
        ParseSwift.initialize(applicationId: "...",
                              clientKey: "...",
                              serverURL: URL(string: "https://parseapi.back4app.com")!,
                              liveQueryServerURL: URL(string: "https://felse.b4a.io")!,
                              allowCustomObjectId: true)
        print("installation is nil: \(Installation.current == nil)")
        return true
    }
```
2) restart app without any activity and observe the `ParseInstallation.current` being nil on second and further launch

### Actual Outcome

A `currentInstallationContainer: CurrentInstallationContainer<Self>` is retrieved from Keychain with set properly `installationId` but  with nil `currentInstallation`
>CurrentInstallationContainer<BaseParseInstallation>(currentInstallation: nil, installationId: Optional("f4ec6107-8d0b-4718-801b-33e97c64c3ea"))

### Expected Outcome

`currentInstallation` should have been saved during the first run

### Environment

Client
- Parse Swift SDK version: 1.9.9
- Xcode version: 12.5
- Operating system (iOS, macOS, watchOS, etc.): iOS
- Operating system version: 14.5.

Server
- Parse Server version: 4.5.0
- Operating system: unknown (back4app service provider)
- Local or remote host (AWS, Azure, Google Cloud, Heroku, Digital Ocean, etc): unknown (back4app service provider)

Database
- System (MongoDB or Postgres): unknown (back4app service provider)
- Database version: `unknown (back4app service provider)
- Local or remote host (MongoDB Atlas, mLab, AWS, Azure, Google Cloud, etc): unknown (back4app service provider)

### Logs

Further description of the issue discussed in [issue 171](https://github.com/parse-community/Parse-Swift/issues/171)
